lib/recording/recording_jobs.ex

```diff
@@ -0,0 +1,150 @@
+defmodule Onvif.Recording.RecordingJobs do
```

Comment:
```suggestion
defmodule Onvif.Recording.RecordingJob do
```
As an embedded schema module this should be singular, same with the filename
Contexts are plural